api.py

```diff
@@ -16,22 +16,27 @@
 UNSUPPORTED_TYPE = 6
 INVALID_VALUE = 7
 
+
 class InjectedTargetError(Exception):
     """Base class for exceptions based on errors returned from injected DLL side"""
     pass
 
+
 class UnsupportedActionError(InjectedTargetError):
     """The specified action is not supported"""
     pass
 
+
 class TargetRuntimeError(InjectedTargetError):
     """Runtime exception during code execution inside injected target"""
     pass
 
+
 class NotFoundError(InjectedTargetError):
```

Comment:
`NotFoundError` sounds too generic. I'd suggest some style for injected package. Maybe start all exception class names with `Injected` ? It's for future PRs.

Reply:
Done in the new `improvements` branch in my fork

Reply:
Please create a PR then. Many small PRs is better than 1 big PR.